apphub/image_classification/pyramidnet/pyramidnet_torch.py

```diff
@@ -0,0 +1,308 @@
+import math
```

Comment:
missing copyright notice

Reply:
I'm not sure we can future-date the notice to 2021. maybe just leave as 2020 for now

Reply:
done